src/ManagerV2/FolderManager.php

```diff
@@ -86,11 +86,15 @@ public function getZipFromFolder(Dossier $folder): ?StreamedResponse
 
     private function createZipFromFolder(Dossier $folder): void
     {
+        $documents = $this->getUser() === $folder->getBeneficiaire()->getUser()
+            ? $folder->getDocuments()
+            : $folder->getPublicDocuments();
+
```

Comment:
`$folder->getDocuments($documents = $this->getUser() === $folder->getBeneficiaire()->getUser())`